aten/CMakeLists.txt

```diff
@@ -218,6 +218,23 @@ ENDIF(UNIX)
 
 INCLUDE (CheckIncludeFile)
 INCLUDE (CheckCSourceCompiles)
+INCLUDE (CheckCSourceRuns)
+
+# Check that our compiler builds and links at all
```

Comment:
Doesn’t CMak do a check like this already? If you set some weird program as CXX it will complain as one of the first steps

Reply:
It checks if compile works, but not run. This matters for `-fsanitize=address`, because running won't work if you don't have `libasan.so` (which is an easy state to be in.)